modules/ding_place2book/ding_place2book.module

```diff
@@ -553,7 +552,7 @@ function ding_place2book_event_op($op, $node, $place2book_id = NULL) {
       $req_result = drupal_http_request($service_settings['service_url'] . '/delete_event', $options);
 
       if ($req_result->status_message != 'Accepted') {
-        $p2b_error = $req_result->headers['status'];
+        $p2b_error = $req_result->headers['error-class'];
```

Comment:
See comment above. Also I find it odd that we refer to `error-class` here but `error-message` above.

The context here seems to indicate that we would want an error message.

Reply:
This was a finding during run-time/debugging. The response headers simply lack the `status` key on those lines and delivered notice errors.
From the implemented logic I assume it tries to get an error string explaining what happened service-wise, so the best value resided in the `error-class` key.

Reply:
The two situations seem very similar. Is there a reason why we would use `error-message` combined with `$req_result->status_message` on L503 and `error-class` (without `$req_result->status_message`) here?

Without any further information I would expect we would use the same key.

Reply:
Good question, since I didn't compare whether the service delivers consistent responses for both actions - `/create_event` and `/delete_event`. In other words, whether these - `status_message`, `error-message` and `error-class` - are present and under what circumstance.